sensor/kubernetes/listener/resource_event_handler.go

```diff
@@ -90,7 +113,21 @@ func (k *listenerImpl) handleAllEvents() {
 		complianceScanInformer = crdSharedInformerFactory.ForResource(complianceoperator.ComplianceScan.GroupVersionResource()).Informer()
 		complianceTailoredProfileInformer = crdSharedInformerFactory.ForResource(complianceoperator.TailoredProfile.GroupVersionResource()).Informer()
 		complianceSuiteInformer = crdSharedInformerFactory.ForResource(complianceoperator.ComplianceSuite.GroupVersionResource()).Informer()
+		// Override the crdHandlerFn to only handle when the resources become unavailable
+		crdHandlerFn = func(status *watcher.Status) {
```

Comment:
[nit] does it make too much of a difference to have two callbacks? Couldn't you define a single callback that checks both available and unavailable?

This was you can define this only once and the `k.pubSub.Publish` call can be done regardless of the status.

Reply:
The problem with having just one callback is that Sensor will enter an infinite loop of restarting the connection because the `watcher` will report the `available` state on start up even if the informers were already started. I could add some complexity to the callback by checking if the informers are running and, if they are, then not restart the connection, but I think it's simpler to just override the callback function. WDYT?